measurement/measures/geometry.py

```diff
@@ -71,6 +71,7 @@ class Distance(AbstractMeasure):
     sears_yard = Unit("0.91439841", ["sears_yd", "Yard (Sears"])
     survey_foot = Unit("0.304800609601", ["survey_ft", "US survey foot", "U.S. Foot"])
     yard = Unit("0.9144", ["yd"])
+    point = Unit(decimal.Decimal("0.0254") / 72, ["pt"])
```

Comment:
```suggestion
    point = Unit(decimal.Decimal("25.4e-3") / 72, ["pt"])
```

Reply:
Thank you for the suggestion, I applied it locally and updated the PR.